aya-obj/src/types.rs

```diff
@@ -0,0 +1,21 @@
+//! Idiomatic Rust wrappers around BPF types.
+
+/// The type of BPF statistic to enable.
+#[non_exhaustive]
+#[doc(alias = "bpf_stats_type")]
+#[derive(Copy, Clone, Debug)]
+pub enum StatsType {
```

Comment:
Is there a reason why this is here and not in the aya crate?

We can't re-export aya-obj symbols as is, because aya-obj is perma-unstable, so
it'd make aya unstable too. In this case I think we can just move the type
definition to the aya crate.

Reply:
> Is there a reason why this is here and not in the aya crate?
> 
> We can't re-export aya-obj symbols as is, because aya-obj is perma-unstable, so it'd make aya unstable too. In this case I think we can just move the type definition to the aya crate.

I was initially thinking that `aya-obj` was used for lower level type bindings on top of the FFI.
`Stats` is now in `aya` crate. 😝

> also nit: I'd call it `Stats` not `StatsType`. I know it matches the underlying kernel name, but in rust seems redundant, we don't have things called `ArcType` `MutexType` etc.

Makes sense. `StatsType` is now `Stats`. 🫡